src/components/Comments.jsx

```diff
@@ -25,31 +35,64 @@ function Comments () {
         })
     },[review_id])
 
-    return (
-    <main>
+    const handleSubmit = (event) => {
+        setFormSubmitted(false)
+        event.preventDefault()
+        console.log(loggedInUser);
+        setNewComment({
+            username: loggedInUser,
+            body: newComment.body
+        })
+        postCommentForReview(review_id, newComment)
+        setFormSubmitted(true)
+    }
+
+     const handleChange = (event) => {
+        //  setNewComment(event.target.value)
+        console.log(event.target.value);
+        const name = event.target.name
+        const value = event.target.value
+        setNewComment(values => ({...values, username: loggedInUser, [name]: value}))
+     }
+
+     return (
+         <main>
           {isLoading ? (
-            <h2>Loading...</h2>
+              <h2>Loading...</h2>
         ) :(
-         <li key={review.review_id} >
-    <article className="individual__review">
-        <img src={review.review_img_url} alt="review img" className="review__img"/>
-        <div className="individual__review__text">
-        <h3>{review.title}</h3>
-        <h4>game designer: {review.designer}</h4>
-        <br/>
-        <h5>review author: {review.owner}</h5>
-        <p>{review.review_body}</p>
-        <p>created at: {review.created_at}</p>
-        <br/>
-        <p>votes: {review.votes}</p>   
-        </div>
+            <li key={review_id} >
+            <article className="individual__review">
+            <img src={review.review_img_url} alt="review img" className="review__img"/>
+            <div className="individual__review__text">
+            <h3>{review.title}</h3>
+            <h4>game designer: {review.designer}</h4>
+            <br/>
+            <h5>review author: {review.owner}</h5>
+            <p>{review.review_body}</p>
+            <p>created at: {review.created_at}</p>
+            <br/>
+            <p>votes: {review.votes}</p>   
+            </div>
     </article>
     </li>
         )}
         <h2>Comments</h2>   
+                    <form className="comment__post" onSubmit={handleSubmit}>
+                        <label htmlFor="username">Username: {loggedInUser} </label>
+                        {/* <input type="text" id="username" name="username" disabled value={newComment.username || ""} onChange={handleChange} /> */}
+                        <label htmlFor="body">Comment: </label>
+                        <input type="text" id="body" name="body" value={newComment.body || ""} onChange={handleChange}/>
```

Comment:
A few things to consider here:
- A textarea is preferred over input, because it allows for multi line comments
- Even if you're not logged in, there is a 'comment submitted' message
- It doesn't post empty comments, but still replies with 'comment submitted'
Check out the 'required' attribute you can put on inputs to help with empty comments
- You should prevent repeat clicks while the comment is still being posted, and then clear the comment box once that post is done